src/components/common/form/ImageCropper.web.js

```diff
@@ -10,7 +11,7 @@ const ImageCropper = props => {
     <Cropper
       ref={cropper}
       src={image}
-      style={{ height: 400, width: '100%' }}
+      style={{ height: getDesignRelativeHeight(400), width: '100%' }}
```

Comment:
percents usage isn't compatible with RN. Should be replaced via calculations relative to the device screen width

Reply:
@serdiukov-o-nordwhale 

Starting from React Native 0.42 height and width accept percentage values.